src/Uploader/PaymentMethodLogoUploader.php

```diff
@@ -72,7 +72,8 @@ private function uploadSingle(MollieGatewayConfigInterface $mollieGatewayConfig)
 
         $this->filesystem->write(
             $customizeImage->getPath(),
-            $file->getPathname()
+            /** @phpstan-ignore-next-line  */
```

Comment:
Could you write comment, why we ignore the line?

Reply:
Fixed to don't ignore it